### DISCUSSION TITLE
DeepL - Automatic Translations

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#15288] [Feature Requests]
I am now going to create a way to auto translate translation-fields in an collection.
Therefore I want to use DeepL. Maybe there is someone already done it.

**Comment by NilsBaumgartner1994** [comment]
Finished
I created a small extension for directus i someone wants to use it
https://github.com/FireboltCasters/directus-extension-auto-translation

**Comment by rijkvanzanten** [comment]
Awesome! Also curious if we can make this happen as a Flow somehow 🤔

**Reply by NilsBaumgartner1994** [reply]
Maybe, I think there is a possibility to add a custom script. If we can access on the auth_key of DeepL (in a collection currently saved :-/ ) and we can run a script from an installed package.

**Reply by NilsBaumgartner1994** [reply]
Currently I like, that it checks new translation jobs for all collections where the 3 fields are defined.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
